test/integration/test_webrtc_api.py

```diff
@@ -0,0 +1,260 @@
+"""
+    Bandwidth
+
+    Bandwidth's Communication APIs  # noqa: E501
+
+    The version of the OpenAPI document: 1.0.0
+    Contact: letstalk@bandwidth.com
+    Generated by: https://openapi-generator.tech
+"""
+
+
+import os
+import unittest
+from array import array
+from urllib import response
+
+from hamcrest import *
+
+import bandwidth
+from bandwidth.api import participants_api, sessions_api
+from bandwidth.exceptions import NotFoundException, UnauthorizedException
+from bandwidth.model.create_participant_request import CreateParticipantRequest
+from bandwidth.model.create_participant_response import CreateParticipantResponse
+from bandwidth.model.publish_permissions_enum import PublishPermissionsEnum
+from bandwidth.model.device_api_version_enum import DeviceApiVersionEnum
+from bandwidth.model.participant import Participant
+from bandwidth.model.session import Session
+from bandwidth.model.subscriptions import Subscriptions
+from bandwidth.model.participant_subscription import ParticipantSubscription
+
+
+class TestSessionsApi(unittest.TestCase):
+    """SessionsApi unit test stubs"""
+
+    def setUp(self):
+        # API Client
+        configuration = bandwidth.Configuration(
+            username = os.environ.get('BW_USERNAME'),
+            password = os.environ.get('BW_PASSWORD')
+        )
+        api_client = bandwidth.ApiClient(configuration)
+        self.sessions_api_instance = sessions_api.SessionsApi(api_client)
+        self.participants_api_instance = participants_api.ParticipantsApi(api_client)
+        self.account_id = os.environ.get('BW_ACCOUNT_ID')
+
+        # Participant Properties
+        self.callback_url = 'https://example.com/callback'
+        self.publish_permissions = [
+                PublishPermissionsEnum('AUDIO'),
+                PublishPermissionsEnum('VIDEO')
+            ]
+        self.participant_tag = 'python integration participant tag'
+        self.device_api_version = DeviceApiVersionEnum('V3')
+        self.participant_id = ''
+
+        # Participant Request
+        self.create_participant_request = CreateParticipantRequest(
+            callback_url=self.callback_url,
+            publish_permissions=self.publish_permissions,
+            tag=self.participant_tag,
+            device_api_version=self.device_api_version
+        )
+
+        # Session Properties
+        self.session_tag = 'python integration session tag'
+        self.session_id = ''
+
+        # Session Request
+        self.session = Session(
+            tag=self.session_tag
+        )
+
+        self.stream_aliases = ['python integration alias']
+
+    def create_participant(self):
+        """Test creating participant
```

Comment:
I think most of these docstrings could be removed altogether. The individual test name is pretty self-explanatory

Reply:
lol my previous comment was to add the docstrings 

lets keep them in there for consistencies sake - IMO its expected

Reply:
haha sounds good to me; it doesn't really make much difference either way